models/org.go

```diff
@@ -280,18 +280,25 @@ type OrgUser struct {
 	UID      int64 `xorm:"INDEX UNIQUE(s)"`
 	OrgID    int64 `xorm:"INDEX UNIQUE(s)"`
 	IsPublic bool  `xorm:"INDEX"`
-	IsOwner  bool
-	NumTeams int
+}
+
+func isOrganizationOwner(e Engine, orgID, uid int64) (bool, error) {
+	ownerTeam := &Team{
+		OrgID: orgID,
+		Name:  ownerTeamName,
+	}
+	if has, err := e.Get(ownerTeam); err != nil {
+		return false, err
+	} else if !has {
```

Comment:
No need for the `else` here, we can just use if since the previous block always returns.

Reply:
IMO, using an `else` is better; we would have to declare `has` before the first `if` statement otherwise.

Reply:
Didn't catch that - fair enough!